scripts/deterministic-slide/convert.py

```
@@ -0,0 +1,756 @@
+
+
+def render_table(table_c: list, images_b64: dict) -> str:
+    """Pandoc Table の AST を簡易な <table> に落とす。
+
+    Pandoc Table の構造 (pandoc-api-version 1.23):
+      [attr, caption, colspecs, head, [bodies], foot]
+    各 row は (attr, [cells])、各 cell は (attr, alignment, rowspan, colspan, [blocks])
+    """
+    # 雑にだけど構造は読み取る
+    try:
+        _attr, caption, _colspecs, head, bodies, foot = table_c
```

Ruff (0.15.20)

[warning] Unpacked variable `foot` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)